bundles/org.openhab.core.io.rest.sitemap/pom.xml

```diff
@@ -24,6 +24,11 @@
       <artifactId>org.openhab.core.ui</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.openhab.core.bundles</groupId>
+      <artifactId>org.openhab.core.test</artifactId>
+      <version>${project.version}</version>
```

Comment:
The scope should be test or do we need that dependency also for the non test relevant stuff?